pkg/ast/pipesearch/searchQueryParser.go

```diff
@@ -450,6 +450,7 @@ func searchPipeCommandsToASTnode(node *QueryAggregators, qid uint64) (*QueryAggr
 			log.Errorf("qid=%d, searchPipeCommandsToASTnode: parseColumnsCmd error: %v", qid, err)
 			return nil, err
 		}
+		pipeCommands.EvalExpr = node.EvalExpr
```

Comment:
Is this useful? I thought the current query pipeline doesn't use `EvalExpr` (I might be wrong). For the new query pipeline, we'll return on line 431 so won't get here

Reply:
I just added it to test the parser, I have also added the test for it [here](https://github.com/siglens/siglens/pull/1786/files#diff-d9d1af5bbb801ffabc34bcc85856933ec8f56a6c629e36bd0ee5bf6ce49e7c7cR11394). Later when we remove the older flow we can change this.

Reply:
I added it to test the parser, have removed it and modified my [test](https://github.com/siglens/siglens/pull/1786/files#diff-d9d1af5bbb801ffabc34bcc85856933ec8f56a6c629e36bd0ee5bf6ce49e7c7cR11394).